lidia/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Can you also mention it here: https://github.com/grafana/pyroscope/blob/main/LICENSING.md